packages/enketo-core/README.md

```diff
@@ -169,6 +169,12 @@ The `language` option overrides the default languages rules of the XForm itself.
 6. run tests with `yarn test` (headless chrome) and `yarn run test-browsers` (browsers); **note:** running tests updates the coverage badge in README.md, but these changes should not be committed except when preparing a release
 7. adding the querystring `touch=true` and reducing the window size allows you to simulate mobile touchscreens
 
+### Common development setup issue troubleshooting
+
+-   `ERESOLVE unable to resolve dependency tree`: Make sure you're using node >=18 <21. Try [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating).
+-   `Error: Cannot find module...` and `npm ERR! code ENOWORKSPACES`: This is a monorepo. Make sure you run `yarn install` in the root of this project before moving into `packages/enketo-core`. If your using nvm, make sure you run `nvm use` from the root of this project before moving into `packages/enketo-core`. Running `nvm use` from `packages/enketo-core` will not work.
```

Comment:
This is admittedly not a regular monorepo contributor, I think that's why I got confused. Maybe this troubleshooting section i suggested could be replaced with some text in the -core package readme that directs people up to the monorepo readme for getting started instructions.

I don't have a strong reason for nvm other than that's what's on my system for use with other projects. Let me take a closer look at volta. Excited to learn about it and see if I can use it side-by-side with my nvm setup!

Reply:
I’m hoping you like Volta!

I agree that each package should make it clear that commands should be run at the top level. 👍

Reply:
Wow, ok, I get it. I'm enjoying Volta. I even uninstalled nvm and am going to try Volta out for node version management across my system :).

That being said, I think nvm accomplishes the same thing for this project, and I wonder if it wouldn't still be helpful to explain that. What do you think of adding something to the root readme that explains: if you use Volta it will pick up the right node version from the package.json, and that if you're not using Volta, you can use your favorite other way of managing node versions (like nvm) to install the version pinned in the package.json.

Reply:
If you're interested in that, I could update this pr to add text like that to the root readme, and a change to the package readmes to point to the root install instructions.

Reply:
Yes, that sounds fine.